aws/cloudformation/vpc.yml.erb

```
@@ -110,6 +116,39 @@ Resources:
     Properties:
       GroupDescription: Enable SSH access to frontends from daemon.
       VpcId: {Ref: VPC}
+  # Used by Application Load Balancers that only serve traffic forwarded to it by CloudFront.
```

```suggestion
  # To be used by Application Load Balancers that only serve traffic forwarded to it by CloudFront.
```

We can update this comment later once it's actually used, without updating the stack.